app/ExposureHistoryContext.tsx

```diff
@@ -87,10 +88,6 @@ const ExposureHistoryProvider: FunctionComponent<ExposureHistoryProps> = ({
     return subscription.remove;
   }, [exposureInfoSubscription, toExposureHistory]);
 
-  useEffect(() => {
-    getCurrentExposures();
-  }, [toExposureHistory]);
-
```

Comment:
@devinjameson can you put this guy back? The calendar doesn't populate with exposures on app launch if this is removed 🙇

Reply:
cc @aledustet